countdownTimer.js

```diff
@@ -1,39 +1,80 @@
-// Set the date we're counting down to
-var countDownDate = new Date("Mar 15, 2022 15:37:25").getTime();
+let timer = "5:00"
+document.getElementById("textfield").innerHTML = timer;
 
-// Update the count down every 1 second
-var x = setInterval(function() {
+var x = false;
+function countDown () {
+ 
+  let currenttime = document.getElementsByTagName('p')[0].textContent;
+  let countdowntime = currenttime.split(":");
+ 
+  let seconds = countdowntime.pop();
+  let minutes = countdowntime.pop();
+  let hours = countdowntime.pop();
+  let days = countdowntime.pop();
 
-  // Get today's date and time
-  var now = new Date().getTime();
+  seconds--;
+ 
+  if (seconds == -1){
```

Comment:
Take a read of:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#strict_equality_using_

In JavaScript you pretty much always want to use `===` instead of `==.`